python/ray/serve/tests/test_controller_recovery.py

```diff
@@ -26,7 +26,8 @@
 from ray.util.state import list_actors
 
 
-def test_recover_start_from_replica_actor_names(serve_instance):
+@pytest.mark.parametrize("autoscaling", [True, False])
```

Comment:
Could you rewrite this as:

```python
@pytest.mark.parametrize("deployment_options", [
    {"num_replicas": 2},
    {"autoscaling_config": {"min_replicas": 2, "max_replicas": 2}},
])
```

That way, we don't need a conditional in the test (since we can pass this directly into the decorator with `**deployment_options`), and the fixture is more generalizable.